### PR TITLE
Create calc.js

### PR DESCRIPTION
**Pull request by optimistguy**
this is the actual command file for the calculator command

### CODE REVIEW

File: commands/calc.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+var math = require('mathjs');
```
Comment (YorkAARGH) [inline commands/calc.js:1]: var? this is 2018, we don't use var

File: commands/calc.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const Discord = require("discord.js");
```
Comment (YorkAARGH) [inline commands/calc.js:2]: Why are you requiring Discord here and on line20?

File: commands/calc.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+exports.run = (bot, message, args, command) => {
```
Comment (YorkAARGH) [inline commands/calc.js:4]: Please follow the variable labelling of the project, we do not use `bot`, we use `client`

File: commands/calc.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    let input = args.join(" ");
```
Comment (YorkAARGH) [inline commands/calc.js:5]: If you're not redefining the variable please change it to const

File: commands/calc.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+    const Discord = require('discord.js');
+    const embed = new Discord.RichEmbed()
+        .setThumbnail("https://images-na.ssl-images-amazon.com/images/I/31QYTepQomL.png")
+        .setColor('RANDOM')
+        .addField("**Question:**", question, true)
+        .addField("**Answer:**", answer)
+        .setFooter(`requested by ${message.author.tag}`);
+
+    message.channel.send({
```
Comment (YorkAARGH) [inline commands/calc.js:28]: This should be on one line

**Review by YorkAARGH — CHANGES_REQUESTED** [review]
These changes should be done to be considered for merging, also please close [PR 66](https://github.com/AnIdiotsGuide/guidebot/pull/66) and commit the package change to this PR.

File: commands/calc.js (hunk 1 above)
Comment (optimistguy) [reply commands/calc.js:1]: I can change it

File: commands/calc.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        embed
+    })
```
Comment (kyranet) [inline commands/calc.js:30]: Missing semicolon-

**Comment by Stitch07** [comment]
Tldr of all those reviews - follow linting rules and you should be fine.

**Comment by Stitch07** [comment]
EBWODP: You should also be stopping the case where a person does `?math sin` and math.eval sends the obscure function for sin.

**Comment by YorkAARGH** [comment]
Closing due to inactivity
